<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1698</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>9</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="editorWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="timeView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>380</y>
      <width>281</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="volView">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>116</width>
      <height>356</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="drawBiff">
       <property name="text">
        <string>Draw Biff</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="BifParamBox"/>
     </item>
     <item>
      <widget class="QComboBox" name="CoordParamBox">
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>    Start Param</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="startParamLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>    End Param</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="endParamLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>    Param Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stepParamLine"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>    Transient Time</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="transientTimeLine"/>
     </item>
    </layout>
   </widget>
   <widget class="QCustomPlot" name="biffGraphView">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lpGraphView">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>380</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="drawButton">
       <property name="text">
        <string>Draw Graph</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="methodBox">
       <item>
        <property name="text">
         <string>Standart Euler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moded Euler</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Euler-Cromer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RungeKutta</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="systemBox">
       <item>
        <property name="text">
         <string>Rossler Attractor</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VanDerPol</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NozeGuver</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>NozeGuverModed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lorenz</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>        N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>     Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stepLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Volume</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="volLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>    x   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>    y    </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>    z   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Volume Step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="volumeStepLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>380</y>
      <width>121</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="lpDrawButton">
       <property name="text">
        <string>LPDraw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lpCoordParamBox">
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>  delta </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="deltaLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>  x </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="xLPLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>  y </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="yLPLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>  z </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="zLPLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="syncGraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>10</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>380</y>
      <width>225</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="syncButton">
       <property name="text">
        <string>Syncronize</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="syncCoordParamBox">
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Master Method</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="masterMethodBox">
         <item>
          <property name="text">
           <string>Standart Euler</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moded Euler</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Euler-Cromer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RungeKutta</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="text">
          <string>Slave Method</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="slaveMethodBox">
         <item>
          <property name="text">
           <string>Standart Euler</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moded Euler</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Euler-Cromer</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>CD</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RungeKutta</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>   k  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="syncKLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>    x   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="syncXLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <item>
        <widget class="QLabel" name="label_20">
         <property name="text">
          <string>    y   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="syncYLine"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>    z   </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="syncZLine"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="yScaleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock y scale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="xScaleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Lock x scale</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="syncTimeGraphView">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>10</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="syncPhaseGraphView">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>380</y>
      <width>271</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1698</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QGraphicsView</extends>
   <header location="global">qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
